Bluedot-Innovation/Bluedot-React-Native-Minimal-Integration
# Po/bd 3045 appcenter crashes

to test:

1. login to the appcenter with the admin account and gather appcenter secrets for both Android and iOS
2. update both `android/app/src/main/assets/appcenter-config.json` and `ios/AppCenter-Config.plist`
2. add `import Crashes from 'appcenter-crashes';` in `components/InitializeSdk.js`
2. add `Crashes.generateTestCrash();` in `function handleInitializeSDK()`
3. start an app and press the init button. You should see crashes coming to AppCenter.
